### PR TITLE
system-tools: include libgpiod

### PR DESCRIPTION
**Pull request by mklein-de**
Tools for interacting with the linux GPIO character device (gpiodetect,
gpioinfo, gpioget, gpioset, gpiofind, gpiomon)

### CODE REVIEW

File: packages/addons/tools/system-tools/package.mk [hunk 1]
```diff
@@ -118,6 +119,10 @@ addon() {
     cp -P $(get_install_dir jq)/usr/bin/jq ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
     cp -P $(get_install_dir oniguruma)/usr/lib/{libonig.so,libonig.so.5,libonig.so.5.2.0} ${ADDON_BUILD}/${PKG_ADDON_ID}/lib
 
+    # libgpiod
+    cp -P $(get_install_dir libgpiod)/usr/bin/{gpiodetect,gpiofind,gpioget,gpioinfo,gpiomon,gpioset} ${ADDON_BUILD}/${PKG_ADDON_ID}/bin
+    cp -L $(get_install_dir libgpiod)/usr/lib/libgpiod.so.2 ${ADDON_BUILD}/${PKG_ADDON_ID}/lib
```
Comment (lrusak) [inline packages/addons/tools/system-tools/package.mk:124]: Could the tools just be built statically or do you think other tools will need the shared libgpiod library?

Comment (mklein-de) [reply packages/addons/tools/system-tools/package.mk:124]: sure, libgpiod can be linked statically. I don't think there many tools using the library.

File: packages/addons/addon-depends/system-tools-depends/libgpiod/package.mk [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2020-present Team LibreELEC (https://libreelec.tv)
```
Comment (chewitt) [inline packages/addons/addon-depends/system-tools-depends/libgpiod/package.mk:2]: 2021-present

**Comment by chewitt** [comment]
@mklein-de looks good, can you squash the commits please

**Review by CvH — APPROVED** [review]
tx for contributing

**Pull request merged**